next.config.js

```diff
@@ -11,7 +11,7 @@ const withMDX = createMDX({
 /** @type {import('next').NextConfig} */
 const nextConfig = {
   pageExtensions: ['ts', 'tsx', 'js', 'jsx', 'md', 'mdx'],
-  reactStrictMode: true,
+  reactStrictMode: false,
```

Comment:
Je ne suis pas sûre que ce soit une bonne idée de passer le reactStrictMode à false. Il me semble que c'est là pour nous faire rendre compte des effets de bord possible. Je me demande d'ailleurs s'il ne faudrait pas plutôt faire l'appel à l'ajout de lignes dans l'excel, au clic sur le dernier "suivant" du formulaire plutôt que dans un useEffect sur la dernière page. Je crois que c'est pas une très bonne pratique 

https://react.dev/learn/synchronizing-with-effects#not-an-effect-buying-a-product